src/pages/index.js

```diff
@@ -7,97 +7,211 @@ import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
-
-//============================== Validation ===========================\\
-const formList = [
-  ...document.querySelectorAll(validationSettings.formSelector),
-];
-
-formList.forEach((form) => {
-  const formValidator = new FormValidator(validationSettings, form);
-
-  formValidator.enableValidation();
+import PopupConfirm from "../components/PopupConfirm.js";
+
+import API from "../components/API.js";
+const api = new API({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c48cc533-37fe-4aec-bc10-a2c7c93d6979",
+    "Content-Type": "application/json",
+  },
 });
 
-//======================= Preview Image popup =====================\\
-
-const previewImagePopup = new PopupWithImage("#preview-image-popup");
-previewImagePopup.close();
-
-//======================= Render initial cards =====================\\
-const cardListEl = document.querySelector(".cards__list");
-
-//create a card instance
+//======================= Create Card Instance =====================\\
 function createCard(data) {
   const cardElement = new Card(
     {
       cardData: data,
       handleCardClick: (cardImage) => {
         previewImagePopup.open(cardImage);
       },
+      handleDeleteSubmit: (cardID) => {
+        deleteCardPopup.setSubmitAction(() => {
+          deleteCardPopup.showLoading();
+          api
+            .deleteCard(cardID)
+            .then(() => {
+              cardElement._handleDeleteButton();
+              deleteCardPopup.close();
+            })
+            .catch((err) => {
+              console.error(err);
+            })
+            .finally(() => {
+              deleteCardPopup.hideLoading();
+            });
+        });
+        deleteCardPopup.open();
+      },
+      handleCardLike: (cardID) => {
+        api
+          .likeCard(cardID)
+          .then(() => {
+            cardElement.updateCardLike(cardData.likes);
+          })
+          .catch((err) => console.error(err));
+      },
+      handleCardUnlike: (cardID) => {
+        api
+          .unlikeCard(cardID)
+          .then(() => {
+            cardElement.updateCardLike(cardData.likes);
+          })
+          .catch((err) => console.error(err));
+      },
     },
     ".card-template_type_default"
   );
+
   return cardElement.generateCard();
 }
 
-//create a section of cards
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (data) => {
-      const cardElement = createCard(data);
-      cardSection.addItem(cardElement);
-    },
+//======================= Render initial cards =====================\\
+const cardListEl = document.querySelector(".cards__list");
+let userID;
+let cardSection;
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([user, initialCards]) => {
+    //set UI
+    userInfo.setUserInfo(user.name, user.about);
+    userID = user._id;
+
```

Comment:
Please create a method `setAvatar` in `UserInfo` class and use it here to set the initial avatar